plugin/ftplugin/neuralbuf.vim

```diff
@@ -0,0 +1,2 @@
+nnoremap <C-CR> :call neural#buffer#RunBuffer()<CR>
```

Comment:
Let's make the keybind configurable.

```vim
call neural#config#Load()
```

I can't think of a very easy way to put the configured key in the command. It may be good to use the `"\<C-CR>"` syntax.